### PR TITLE
Verification

### PR DESCRIPTION
**Pull request by jasonmotha**

### CODE REVIEW

File: scripts/management/slurm_query_status.py [hunk 1]
```diff
@@ -105,12 +106,14 @@ def check_dependancy_met(task, task_list):
...
 
 def get_runnable_tasks(db, n_runs=N_TASKS_TO_RUN, retry_max=RETRY_MAX):
+    do_verification = False
     db_tasks = get_db_tasks_to_be_run(db, retry_max)
     tasks_to_run = []
     for task in db_tasks:
         status = task[2]
         if status == 'created' and check_dependancy_met(task, db_tasks):
-            tasks_to_run.append(task)
+            if task < Process.rrup or do_verification:
```
Comment (claudio525) [inline scripts/management/slurm_query_status.py]: Is it possible for do_verification to be True?

File: scripts/cybershake/auto_submit.py [hunk 2]
```diff
@@ -19,79 +20,112 @@
...
-        if binary_mode == True:
-            #update the mgmt_db
-            update_mgmt_db.update_db(db,'winbin_aio','completed',run_name=run_name)    
+
+    lf_sim_dir = os.path.join(sim_dir, "LF/%s" % run_name)
+    params_uncertain_path = os.path.join(lf_sim_dir, "params_uncertain.py")
+    if proc_type == Process.EMOD3D.value:
+        check_params_uncertain(params_uncertain_path)
+        cmd = "python $gmsim/workflow/scripts/submit_emod3d.py --auto --srf %s" % run_name
+        print cmd
```
Comment (claudio525) [inline scripts/cybershake/auto_submit.py]: use print as function?

File: scripts/cybershake/auto_submit.py [hunk 3]
```diff
@@ -19,79 +20,112 @@
...
+            print cmd
+            call(cmd, shell=True)
+
+        if proc_type == Process.Empirical.value:
+            cmd = "$gmsim/workflow/scripts/submit_empirical.py -np 40 -i {} {}".format(run_name, mgmt_db_location)
+            print cmd
+            call(cmd, shell=True)
+
+        if proc_type == Process.Verification.value:
+            pass
```
Comment (claudio525) [inline scripts/cybershake/auto_submit.py]: I assume this is meant to do nothing for now?

File: scripts/management/slurm_query_status.py [hunk 4]
```diff
@@ -84,10 +84,11 @@ def is_task_complete(task, task_list):
 def check_dependancy_met(task, task_list):
     process, run_name, status = task
     process = db_helper.Process(process)
-    if process is Process.EMOD3D or process is Process.HF:
+    if process in (Process.EMOD3D, Process.HF, Process.rrup):
```
Comment (claudio525) [inline scripts/management/slurm_query_status.py]: Should Process.Verification be in one of these if statements?

File: scripts/management/slurm_query_status.py (hunk 1 above)
Comment (jasonmotha) [reply scripts/management/slurm_query_status.py]: Currently not. But will be extended to be an input from somewhere (potentially argparse or calling function)

File: scripts/cybershake/auto_submit.py (hunk 3 above)
Comment (jasonmotha) [reply scripts/cybershake/auto_submit.py]: Exactly.

File: scripts/management/slurm_query_status.py (hunk 4 above)
Comment (jasonmotha) [reply scripts/management/slurm_query_status.py]: Yupp, but as you said currently it does nothing for now.

**Comment by claudio525** [comment]
Closing, as these changes will be in my new PR

**Pull request closed without merging**